orm/src/test/java/org/hibernate/search/test/util/DdlTransactionIsolatorTestingImpl.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test.util;
+
+import java.sql.Connection;
+
+import org.hibernate.dialect.Dialect;
+import org.hibernate.engine.jdbc.connections.spi.ConnectionProvider;
+import org.hibernate.engine.jdbc.connections.spi.JdbcConnectionAccess;
+import org.hibernate.engine.jdbc.spi.JdbcServices;
+import org.hibernate.engine.jdbc.spi.SqlExceptionHelper;
+import org.hibernate.engine.jdbc.spi.SqlStatementLogger;
+import org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl;
+import org.hibernate.service.ServiceRegistry;
+import org.hibernate.tool.schema.internal.exec.JdbcConnectionAccessConnectionProviderImpl;
+import org.hibernate.tool.schema.internal.exec.JdbcConnectionAccessProvidedConnectionImpl;
+import org.hibernate.tool.schema.internal.exec.JdbcContext;
+
+/**
+ * Copied from Hibernate ORM: being originally part of the `org.hibernate.test.util` package,
+ * this is not published in a jar which could be consumed by Hibernate Search.
+ * This copy is not meant to stay here for long.
```

Comment:
I agreed with Steve that we should move the `DdlTransactionIsolatorTestingImpl.java` file to ORM testing utilities, so we'll be able to depend on it. Not least, it means they'll fix the file in case there's need for further changes ;)

So I prefer to use a plain copy of their class, to that now I know which class exactly I need to move.